CMakeLists.txt

```diff
@@ -1104,6 +1105,18 @@ IF (PerlModular)
 	ENDIF()
 ENDIF()
 
+# Scala modular
+IF (ScalaModular)
+	FIND_PACKAGE(Scala REQUIRED)
+	UNSET(TARGET_SWIGFLAGS)
+	IF(EXISTS ${CMAKE_SOURCE_DIR}/src/interfaces/java_modular)
+		IF(!JavaModular)
```

Comment:
Yeah, cause scala can directly use Java files. So I think we don't need another modular for Scala